src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +16,16 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        if (numbers == null || numbers.size() == 0) {
+            throw new RuntimeException("Can't get min value from list:");
+        }
+        return numbers.stream()
+                .map(n -> n.split(","))
+                .flatMap(s -> Arrays.stream(s))
+                .map(p -> Integer.parseInt(p))
```

Comment:
```suggestion
                .map(n -> n.split(","))
                .flatMap(s -> Arrays.stream(s))
                .map(p -> Integer.parseInt(p))
```
В рекомендаціях вказано по можливості не використовувати  послідовні map. Строчки 23-24 можна об'єднати в одну flatMap .